context/app/static/js/components/detailPage/SummaryItem.jsx

```diff
@@ -1,31 +1,41 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import styled from 'styled-components';
 import Typography from '@material-ui/core/Typography';
 import Divider from '@material-ui/core/Divider';
 
 const VerticalDivider = styled(Divider)`
   margin-left: ${(props) => props.theme.spacing(1)}px;
-  margin-right: ${(props) => props.theme.spacing(1)}px;
   height: 15px;
   background-color: ${(props) => props.theme.palette.text.primary};
   align-self: center;
 `;
 
 const StyledDiv = styled.div`
   display: flex;
+  margin-right: ${(props) => props.theme.spacing(1)}px;
 `;
 
-function SummaryItem(props) {
-  const { children, statusIcon } = props;
+function SummaryItem({ children, statusIcon, showDivider }) {
   return (
     <StyledDiv>
       {statusIcon}
       <Typography variant="h6" component="p">
         {children}
       </Typography>
-      <VerticalDivider orientation="vertical" flexItem />
+      {showDivider && <VerticalDivider orientation="vertical" flexItem />}
```

Comment:
I think it would be simpler just to pull this out, and we'll explicitly give `<VerticalDivider orientation="vertical" flexItem />`. We're not saving many lines of code with this.